quelpa.el

```diff
@@ -423,6 +429,10 @@ Passing the option is necessary on the systems where the default
 tar format isn't gnu."
   :type 'boolean)
 
+(defvar quelpa-build-bsdtar-p -1
+  "Current `quelpa-build-tar-executable' is bsdtar or not.
+-1 means this variable is not yet initialized.")
```

Comment:
I wondered about this with the `-1` but I saw further down why you are doing it. I don't know but I think there could be a better way to deal with this instead of using `-1` in this way. Maybe @alphapapa has an idea?

Reply:
It would probably be better to have a variable named something like `quelpa-tar-executable-type` whose value would be, e.g. `gnu` or `bsd`, and if it were nil, that would mean that its type had not yet been determined.

Reply:
yeah, that sounds better to me. I think emacs doesn't have functions to create tar archives. it does have `tar-mode` which can manipulate existing tar archives which quelpa is also using in other places. but if there is anything native like that it would of course be better to use it.

Reply:
Thanks, updated.